src/extra/Markdown/index.tsx

```diff
@@ -1,743 +1,81 @@
 import * as React from 'react';
-import marked from 'marked';
-import highlightJS from 'highlight.js';
-import 'highlight.js/styles/default.css';
-import sanitizeHtml from 'sanitize-html';
-import styled from 'styled-components';
 import Txt, { TxtProps } from '../../components/Txt';
-import { parseMarkdown } from '../utils';
```

Comment:
We need to remove everything in utils that is not used anymore, and remove the unused packages which add up a lot to the bundle

Reply:
Sure 👍